_posts/2017-4-12-swarm-service-discovery.markdown

```diff
@@ -0,0 +1,209 @@
+---
+layout: post
+title: "Service Discovery under Docker Swarm Mode"
+date:   2017-04-12
+author: "@ajeetsraina"
+tags: [linux,operations]
+categories: intermediate
+img: "swarm-service-discovery.png"
+terms: 2
+---
+Service Discovery under Swarm Mode.
+
+## Purpose
+
+The purpose of this lab is to illustrate how Service Discovery works under Swarm Mode.
+
+## The application
+
+WordPress is an open-source content management system (CMS) based on PHP and MySQL. 
+It is a very simple containers application often used for demo purposes during meetup and conferences.
+
+
+## Init your swarm
+
+Let's create a Docker Swarm first. Open up the first instance and initiate Swarm mode cluster.
+
+```.term1
+docker swarm init --advertise-addr $(hostname -i)
+```
+
+This node becomes a master node. The output displays a command to add a worker node to this swarm as shown below:
```

Comment:
Is it relevant to the content of the article how to add a worker?. It's not mentioned anywhere else.